README.md

```diff
@@ -71,28 +86,26 @@ All image versions can be found at: https://hub.docker.com/r/bcsecurity/empire/
 
 ## Quickstart
 
-Check out the [Empire wiki](https://github.com/EmpireProject/Empire/wiki/Quickstart) for instructions on getting started with Empire.
+Check out the [Empire wiki](https://github.com/BC-SECURITY/Empire/wiki/Quickstart) for instructions on getting started with Empire.
 
 ## To Do List
 
-* ~~Port code to work with Python 3~~
 * [Invoke-SocksProxy](https://github.com/p3nt4/Invoke-SocksProxy)
-* Function name randomization
-* ~~JA3/S signature randomization~~
-* ~~Multi-menu function calls~~
-* Function name aliasing
-* ~~Update to [Mimikatz 2.2.0](https://github.com/gentilkiwi/mimikatz)~~
+* Python 2.7 launchers calling back to Python 3.x
```

Comment:
We should consider removing the Todo list here. Just create issues for things in the pipeline

Reply:
Going to leave it for now.